test/benchmark/tests/utils/apache_bench.py

```diff
@@ -66,20 +66,17 @@ def run_apache_bench(self, requests, concurrency, input_file):
         else:
             self.connection.run(f"cp {input_file} {os.path.join(TMP_DIR, 'benchmark/input')}")
 
+        # Run warmup
+        apache_bench_warmup_command = f"ab -c {concurrency} {100} -k -p {TMP_DIR}/benchmark/input -T application/jpg {self.inference_url}/predictions/benchmark"
```

Comment:
I may be going blind but could you explain how the warmup is done? 

Are you doing 100 requests before starting the rest of the count?

Reply:
@nskool let figure out the impact of keeping the connection alive (-k flag) on the overall latency as @lxning mentioned as well.

Reply:
@msaroufim 

Yes, it's doing a 100 iterations before the rest of the count is done. This particular PR assumes that we're trying to discount the higher latency observed when initially loading the model. 

However, separately , I'm exploring if we can discount the first several requests in a single set of consecutive requests that are used to benchmark the model server.

@HamidShojanazeri 
In case of BERT, not specifying `-k` gives me an error:
```
warmup command used: ab -c 100 -n 100 -p /home/ubuntu/benchmark/input -T application/jpg http://127.0.0.1:8080/predictions/benchmark
apache bench command used: ab -c 100 -n 10000 -p /home/ubuntu/benchmark/input -T application/jpg http://127.0.0.1:8080/predictions/benchmark > /home/ubuntu/benchmark/result.txt
apr_pollset_poll: The timeout specified has expired (70007)

apache bench command failed.
```